crates/api_models/src/payments.rs

```diff
@@ -475,6 +475,9 @@ pub enum BankDebitData {
         /// Routing number for ach bank debit payment
         #[schema(value_type = String, example = "110000000")]
         routing_number: Secret<String>,
+
+        #[schema(value_type = String, example = "John Test")]
+        bank_account_holder_name: Secret<String>,
```

Comment:
is this mandatory for all connector? stripe was implemented without this field right?